x-pack/plugins/ml/public/settings/filter_lists/list/header.js

```diff
@@ -64,10 +81,22 @@ export function FilterListsHeader({ totalCount, refreshFilterLists }) {
       <EuiText>
         <p>
           <EuiTextColor color="subdued">
-            Filter lists contain values that you can use to include or exclude events from the machine learning analysis.
-            You can use the same filter list in multiple jobs.
-            <br/>
-            <EuiLink href={docsUrl} target="_blank">Learn more</EuiLink>
+            <FormattedMessage
+              id="xpack.ml.settings.listHeader.filterListsContainsNotAllowedValuesDescription"
+              defaultMessage="Filter lists contain values that you can use to include or exclude events from the machine learning analysis.
+You can use the same filter list in multiple jobs.{br}{learnMoreLink}"
+              values={{
+                br: <br />,
```

Comment:
i think you should use html prefix here

Reply:
no, html prefix is necessary only for angular directive